gemini/tuning/supervised_finetuning_using_gemini.ipynb

```diff
@@ -419,7 +489,137 @@
       },
       "source": [
         "- Data used in this notebook is present in the public Google Cloud Storage(GCS) bucket.\n",
-        "- Validation data should be less than or equal to 250 samples."
+        "- It's in Gemini 1.0 dataset format for finetuning"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "aV_GZg_LaNmV"
+      },
+      "outputs": [],
+      "source": [
+        "!gsutil ls gs://github-repo/generative-ai/gemini/tuning/summarization/wikilingua"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "tKupp3g6W1XS"
+      },
+      "source": [
+        "#### Download the dataset"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "b-EQ4FcExIfp"
+      },
+      "outputs": [],
+      "source": [
+        "!gsutil cp gs://github-repo/generative-ai/gemini/tuning/summarization/wikilingua/* ."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "6wmBkAUoyzdJ"
+      },
+      "source": [
+        "#### Convert Gemini 1.0 tuning dataset to Gemini 1.5 tuning dataset format"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "7bhtxKbha_wS"
+      },
+      "outputs": [],
+      "source": [
+        "def save_jsonlines(file, instances):\n",
+        "    \"\"\"\n",
+        "    Saves a list of json instances to a jsonlines file.\n",
+        "    \"\"\"\n",
+        "    with jsonlines.open(file, mode=\"w\") as writer:\n",
+        "        writer.write_all(instances)"
+      ]
+    },
```

Comment:
For JSONL I/O I recommend using Pandas to directly download/output to GCS. I find it easier/cleaner to work with. (If it works for this particular transformation)

Example: https://github.com/GoogleCloudPlatform/generative-ai/pull/1138#discussion_r1767113650

Reply:
Thanks for the suggestion. 
`instances` is a list of json objects, and there is no use of dataframe that's why using `jsonlines`

Reply:
For next time, will keep it in mind, thanks